.travis.yml

```diff
@@ -11,8 +11,8 @@ python:
 
 env:
   - VERSION="8.0" LINT_CHECK="1"
-  - VERSION="8.0" ODOO_REPO="odoo/odoo" LINT_CHECK="0" INCLUDE="woocommerceerpconnect"
-  - VERSION="8.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0" INCLUDE="woocommerceerpconnect"
+  - VERSION="8.0" ODOO_REPO="odoo/odoo" LINT_CHECK="0" INCLUDE="connector_woocommerce"
```

Comment:
If I remove "include" section, I guess it will not install "connector_woocommerce" module into the instance. Correct me if I am wrong..